simplq/src/components/pages/Admin/TourSteps.jsx

```diff
@@ -70,3 +71,28 @@ export const hasUserBeenOnTour = () => {
   }
   return false;
 };
+
+  export const stepChange = (stepNumber) => {
+    if(stepNumber == 0)
+    {
+      let leftArrow = document.querySelector("[data-tour-elem='left-arrow']");
+      let rightArrow = document.querySelector("[data-tour-elem='right-arrow']");
+      if(leftArrow && rightArrow)
+      {
+        leftArrow.childNodes[0].style.color = "grey";
+        rightArrow.childNodes[0].style.color = "white";
+      }
+    }
+    else if(stepNumber == 1)
+    {
+      let leftArrow = document.querySelector("[data-tour-elem='left-arrow']");
+      let rightArrow = document.querySelector("[data-tour-elem='right-arrow']");
+
+      if(leftArrow && rightArrow)
+      {
+        leftArrow.childNodes[0].style.color = "white";
+        rightArrow.childNodes[0].style.color = "grey";
+      }
+    }
+  }
+  
```

Comment:
```
export const stepChange = (stepNumber) => {
  const leftArrow = document.querySelector("[data-tour-elem='left-arrow']");
  const rightArrow = document.querySelector("[data-tour-elem='right-arrow']");

  if(leftArrow && rightArrow) {
    if(stepNumber == 0) {
        leftArrow.childNodes[0].style.color = "grey";
        rightArrow.childNodes[0].style.color = "white";
    } else { // not else if
        leftArrow.childNodes[0].style.color = "white";
        rightArrow.childNodes[0].style.color = "grey";
  }
}
```

Also curious, why is `if(leftArrow && rightArrow)` required?

Reply:
The [data-tour-elem='left-arrow'] & [data-tour-elem='right-arrow'] are the selectors of reatTour package(third party).
Tomorrow, if the reactTour arrow's selectors are changed to something other than this, then I don't want this part of code causing issue in future.

Reply:
Ok. Lets then add an else clause:
```
import * as Sentry from '@sentry/react';

if(leftArrow && rightArrow) {
   ...
} else { 
   Sentry.captureMessage("left-arrow or right-arrow selectors of reatTour package not found");
}
```

Reply:
Done.

Reply:
Thanks